packages/admin-panel/src/VizBuilderApp/components/PreviewSection.js

```diff
@@ -114,7 +126,11 @@ export const PreviewSection = ({ enabled, setEnabled }) => {
   const columns = useMemo(() => getColumns(reportData), [reportData]);
   const data = useMemo(() => reportData, [reportData]);
 
-  const viewContent = { data, ...visualisation.presentation };
+  // only update Chart Preview when play button is clicked
+  useEffect(() => {
+    const newViewContent = { data, ...visualisation.presentation };
+    setViewContent(newViewContent);
```

Comment:
Like Rohan mentioned, there are pros and cons to this but it's well implemented either way.

Reply:
I am keen on this change overall, but it does really seem to suggest that the 'play' button should somewhere other than inside the report config section... perhaps in the preview header area?

Reply:
(Not suggesting we change that now btw, let's get Dave to take a look at it at some point 👍)

Reply:
Yep let the design master figure it out :)